omniauth-dsds.gemspec

```diff
@@ -20,6 +20,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "multi_json", "~> 1.11.0"
   spec.add_runtime_dependency "omniauth-oauth2", "~> 1.2.0"
 
-  spec.add_development_dependency "bundler", "~> 1.8"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "dotenv", "~> 2.0.0"
```

Comment:
do we need to use dotenv here? its an extra dependency that can easily be done without

in tests can just stub the ENV object
```
allow(ENV).to receive(:fetch).with('BLAH').and_return('FOO')
```